docs/content/documentation/deployment/github-pages.md

```diff
@@ -43,6 +43,7 @@ name: Build and deploy GH Pages
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
can we just point out that this branch name might change depending on the person in the docs below?

Reply:
Sorry about the delayed response! Let me know if the second commit is sufficient 🙂